Bingo_CSD_412/Models/Category.cs

```diff
@@ -10,12 +10,62 @@ public class Category
         //static category TODO: have to implement in Database 
         public static readonly Category CSD_412_Category = new Category();
         public String Name { get; set; }
-        public String[] Words { get; set; }
+        public HashSet<string> Words { get; set; }
+        public int Id { get; }
 
-        static Category()
+        static Category() //Kept this for testing purposes
         {
             CSD_412_Category.Name = "CSD_412";
-            CSD_412_Category.Words = new[] { "HTTP" , "URL", "Data Tier", "N Tier", "Application Settings" };
+            CSD_412_Category.Words.Add("HTTP");
+            CSD_412_Category.Words.Add("URL");
+            CSD_412_Category.Words.Add("Date Tier");
+            CSD_412_Category.Words.Add("N Tier");
+            CSD_412_Category.Words.Add("Application Settings");
+        }
+
+        /*public Category()  <-- Real constructor
+        {
+            Id = GenerateId();
+            Name = GetName();
+            Words = new HashSet<string>();
+        }*/
+
+        private string GetName()
+        {
+            //TODO: Add funcationality that allows user to name their category
+            //Return that user input as a String
+            throw new NotImplementedException();
+        }
+
+        private int GenerateId()
+        {
+            //TODO: Retrive from database the count of categories, and increament by one
+            //return CategoryCount++;
+            throw new NotImplementedException();
+        }
+
+
+        //TODO: View must implement a GUI for the user to send information to the Model
+        public void AddWord(string word) //TODO: Implement an update to database
+        {   
+            Words.Add(word);
+        }
+
+        public void EditWord(string oldword, string newword) //TODO: Implement an update to database
```

Comment:
I think SwapWord is more descriptive.
And also, this functionality can be achieved by user by first deleting an old word, and then adding a new word.

Reply:
Good eye this method is redundant